lib/editor-features.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Experimental editor features config functionality.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Returns the default config for editor features,
+ * or an empty array if none found.
+ *
+ * @return array Default features config for the editor.
+ */
+function gutenberg_experimental_get_editor_features_config() {
+	$empty_config = array();
+	$config_path  = __DIR__ . '/experimental-default-theme.json';
```

Comment:
As you suggested, I'm thinking that we can move the `theme.json` (both core and theme) to PHP. This also solves one potential code path in which we return the empty config without any values. We can do this in a follow-up as we also have to do it for global styles.

Reply:
Let's take care of it separately. I'm still unsure whether this JSON file shouldn't be used as a default for `@wordpress/block-editor` settings object:

https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/src/store/defaults.js

There is also this question whether should I put some defaults there as well for features 😅